src/AppBundle/Event/EventInvitation.php

```diff
@@ -41,8 +42,69 @@ class EventInvitation
      */
     public $guests = [];
 
+    public static function createFromAdherent($adherent = null): self
+    {
+        $dto = new self();
+
+        if ($adherent instanceof Adherent) {
```

Comment:
Please remove this check and type hint the parameter, also considering the name of the method it should not be nullable.